django_babel/extract.py

```diff
@@ -32,7 +32,14 @@ def extract_django(fileobj, keywords, comment_tags, options):
     encoding = options.get('encoding', 'utf8')
     text = fileobj.read().decode(encoding)
 
-    for t in Lexer(text, None).tokenize():
+#    try:
+#        text_lexer = Lexer(text)
+#    except TypeError:
+        # Django 1.9 changed the way we invoke Lexer; older versions
+        # require two parameters.
+    text_lexer = Lexer(text, None)
```

Comment:
I'm getting confused because this line here should fail when run on Django 1.9, but it's not: https://travis-ci.org/python-babel/django-babel/builds/94767535

I'm going to spend some time experimenting. I worry that things aren't actually being run properly on the specified versions of Django and Python.